compliance/index/v4/convert_test.go

```diff
@@ -0,0 +1,303 @@
+package v4
+
+import (
+	"testing"
+
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoassert"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestIndexReportConvert(t *testing.T) {
+	suite.Run(t, new(indexReportConvertSuite))
+}
+
+type indexReportConvertSuite struct {
+	suite.Suite
+}
+
+func (s *indexReportConvertSuite) TestToNodeInventory() {
+	r := createVulnerabilityReport()
+
+	actual := ToNodeScan(r)
+
+	s.Equal(storage.NodeScan_SCANNER_V4, actual.GetScannerVersion())
+	s.Equal(1, len(actual.GetComponents()))
+	s.Equal("openssh-clients", actual.GetComponents()[0].GetName())
+	s.Equal("8.7p1-38.el9", actual.GetComponents()[0].GetVersion())
+	s.Equal("RHSA-2024:4616", actual.GetComponents()[0].GetVulns()[0].GetCve())
+	s.Equal([]storage.NodeScan_Note{storage.NodeScan_UNSUPPORTED}, actual.GetNotes())
+}
+
+func (s *indexReportConvertSuite) TestEmptyReportConversionNoPanic() {
```

Comment:
The test name suggests that maybe we should use `assert.NotPanics` as the assertion

Reply:
Great idea, done!